dynamic/coinchange.go

```diff
@@ -0,0 +1,29 @@
+package dynamic
+/* 
+@params 
+coins: combination of coin 
+amount: target coins
+
+@return
+num of combination
+ */
+
+
+func coinChange(coins []int32, amount int32) int32 {
```

Comment:
Export this function:
```suggestion
func CoinChange(coins []int32, amount int32) int32 {
```